src/sentry/models/integration.py

```diff
@@ -13,6 +13,20 @@
 from sentry.signals import integration_added
 
 
+class PagerDutyServiceProject(Model):
+    __core__ = False
+
+    project = FlexibleForeignKey("sentry.Project", db_index=False, db_constraint=False)
+    organization_integration = FlexibleForeignKey("sentry.OrganizationIntegration")
+    integration_key = models.CharField(max_length=255)
```

Comment:
Do we need an index on any of these columns? Not sure if we're likely to query on these at all.

Reply:
I only really see us using the organizationIntegration and the project to get the other values